### PR TITLE
Add logger

### PR DESCRIPTION
**Pull request by Savlik**

### CODE REVIEW

File: t/01_upload_download.t [hunk 1]
```diff
@@ -30,6 +30,7 @@ my $cfg = {
         host       => 'host',
     },
     s3_enabled => 0,
+    log => []
```
Comment (luckamohelnikova) [inline t/01_upload_download.t:33]: nevim, jestli bych tu nedala log na screenu.

**Comment by luckamohelnikova** [comment]
za me :+1:

**Review by luckamohelnikova — COMMENTED** [review]
:+1:

File: lib/Stor.pm [hunk 2]
```diff
@@ -113,11 +113,17 @@ sub get_from_s3 ($self, $c, $sha) {
...
+    $tx->res->content->unsubscribe('read')->on(
         read => sub {
             my (undef, $chunk) = @_;
             if ($chunk) {
-                $c->write($chunk);
+                try {
+                    $c->write($chunk);
+                }
+                catch{
+                    $c->app->log->warning('Writing chunk failed: $@');
```
Comment (JaSei) [inline lib/Stor.pm]: In single quotes `' '` variable isn't interpolted

**Review by JaSei — CHANGES_REQUESTED** [review]

**Pull request merged**